ci/images/ci-runner/hack/manifests/sidecar/kustomization.yaml

```diff
@@ -2,6 +2,8 @@
 apiVersion: kustomize.config.k8s.io/v1beta1
 kind: Kustomization
 
+namespace: default
```

Comment:
Checkov complains about not defining a specific namespace in the manifest and defining a ns as `default` in the manifests is also not a good practice, that's why kustomizing the ns solves us that problem.

Reply:
In `pod.yaml` and `pvc.yaml` we specify the namespace as `ci-runner` because Checkov does not like when the `default` namespace is used.  However in the `kustomization.yaml` we override that value to `default` so we can keep the current behavior.

If that's correct, you can resolve the conversation.